neo/api/JSONRPC/JsonRpcApi.py

```diff
@@ -409,3 +419,60 @@ def list_address(self):
                 "watchonly": addr.IsWatchOnly,
             })
         return result
+
+    def send_many(self, params):
+        if len(params) not in [1, 2, 3] or type(params[0]) is not list:
```

Comment:
`type(params[0])` will throw a list index out of bounds if `params` is `None` because the first check will fail and then it will still try this because of the `or`

Reply:
@ixje Fixed. Please see code as it appears here:
https://github.com/jseagrave21/neo-python/blob/47fa65d6bc3d446da8a63fa2cfb57ab1ee599d77/neo/api/JSONRPC/JsonRpcApi.py#L560